src/Core/src/Handlers/Entry/EntryHandler.Windows.cs

```diff
@@ -64,8 +66,11 @@ protected override void DisconnectHandler(TextBox platformView)
 		public static void MapHorizontalTextAlignment(IEntryHandler handler, IEntry entry) =>
 			handler.PlatformView?.UpdateHorizontalTextAlignment(entry);
 
-		public static void MapVerticalTextAlignment(IEntryHandler handler, IEntry entry) =>
-			handler.PlatformView?.UpdateVerticalTextAlignment(entry);
+		public static void MapVerticalTextAlignment(IEntryHandler handler, IEntry entry)
+		{
+			if (handler is EntryHandler entryHandler)
+				handler.PlatformView?.UpdateVerticalTextAlignment(entry, entryHandler._placeholder);
```

Comment:
Is it possible to find the placeholder via the name or view apis? Then we don't need to cast and avoid concrete types in these mappers.